pennylane/devices/default_tensor.py

```diff
@@ -708,6 +727,9 @@ def _get_measurement_function(
             if isinstance(measurementprocess, ExpectationMP):
                 return self.expval
 
+            if isinstance(measurementprocess, StateMP):
```

Comment:
Thanks for this! Can we update the documentation to state that this measurement type is also accepted?